bim_webviewer/bim_web_viewer.md

```diff
@@ -1741,7 +1768,9 @@ Retrieves a JavaScript object that describes the object.
 
 ##### Returns
 
-See [Meshnode](#meshnode).
+```js
+Promise<any | null> 
+```
```

Comment:
`suggestion (non-blocking):` This seems like it's gotten less informative. Why not still link to `Meshnode`?

```md
\```js
Promise<Meshnode | null> 
\```

See [Meshnode](#meshnode).
```

`\` is just to get around the formatting in this comment, but hopefully the suggestion is clear.

Reply:
Oh hell yae, I totally forgot we had this.